vllm/_custom_ops.py

```diff
@@ -1135,6 +1135,53 @@ def scaled_fp4_experts_quant(
     return output, output_scales
 
 
+# Using the default value (240.0) from pytorch will cause accuracy
+# issue on dynamic quantization models. Here use 224.0 for rocm.
+FP8_DTYPE = current_platform.fp8_dtype()
+FP8_MAX = 224.0 if current_platform.is_rocm() else torch.finfo(FP8_DTYPE).max
```

Comment:
Is this universal on ROCm or just for the fnuz type? I thought it was the latter

Reply:
I'm just copying the behavior from the reference implementation that I assume was added by AMD folks